rma/models/rma.py

```diff
@@ -575,12 +579,15 @@ def _send_draft_email(self):
 
     def _send_confirmation_email(self):
         """Auto send notifications"""
-        for rma in self.filtered(lambda p: p.company_id.send_rma_confirmation):
+        subtype = self.env.ref("rma.mt_rma_notification")
```

Comment:
The same. Useless change. I know that it seems more performant, but it's less readable and the performance penalty thanks to the cache is barely.

Reply:
I don't agree. It's cleaner and better

Reply:
Explain why. Anyway, what we want is to reduce the diff, so for better chances to get this merged, this PR should touch only the significant parts. For refactorings, do another PR.

Reply:
I agree, this change is not linked to the commit and PR.

Cleaner as it is looping once on self instead of twice